tests/org.jnario.lib.tests/src/org/jnario/lib/tests/ShouldTest.xtend

```diff
@@ -0,0 +1,1035 @@
+		val obj1 = "abcdef"
+		val obj2 = true
+		val obj3 = true as Object
+		val obj4 = false
+		val obj5 = false as Object
+		should_be(obj1, true).assertTrue
+		should_be(obj1, false).assertFalse
+		should_be(obj2, true).assertTrue
+		should_be(obj2, false).assertFalse
+		should_be(obj3, true).assertTrue
+		should_be(obj3, false).assertFalse
+		should_be(obj4, true).assertFalse
+		should_be(obj4, false).assertTrue
+		should_be(obj5, true).assertFalse
+		should_be(obj5, false).assertTrue
+	}
+
+	@Test
+	def should_be_Tboolean_1() {
+		should_be(null as Boolean, true).assertTrue
+		should_be(null as Boolean, false).assertFalse
+	}
+
+	@Test
+	def operator_doubleArrow_Tboolean_0() {
+		val obj1 = "abcdef"
+		val obj2 = true
+		val obj3 = true as Object
+		val obj4 = false
+		val obj5 = false as Object
+		operator_doubleArrow(obj1, true).assertTrue
+		operator_doubleArrow(obj1, false).assertFalse
+		operator_doubleArrow(obj2, true).assertTrue
+		operator_doubleArrow(obj2, false).assertFalse
+		operator_doubleArrow(obj3, true).assertTrue
+		operator_doubleArrow(obj3, false).assertFalse
+		operator_doubleArrow(obj4, true).assertFalse
+		operator_doubleArrow(obj4, false).assertTrue
+		operator_doubleArrow(obj5, true).assertFalse
+		operator_doubleArrow(obj5, false).assertTrue
+	}
+
+	@Test
+	def operator_doubleArrow_Tboolean_1() {
+		operator_doubleArrow(null as Boolean, true).assertTrue
+		operator_doubleArrow(null as Boolean, false).assertFalse
+	}
+
+	@Test
+	def should_startWith_CharSequenceString_0() {
+		val obj1 = "abcdef"
+		val obj2 = "abcdef"
+		val obj3 = "ghi"
+		val obj4 = "ac"
+		val obj5 = "ab"
+		should_startWith(obj1, obj1).assertTrue
+		should_startWith(obj1, obj2).assertTrue
+		should_startWith(obj1, obj3).assertFalse
+		should_startWith(obj1, obj4).assertFalse
+		should_startWith(obj1, obj5).assertTrue
+	} 
+
+	@Test
+	def should_startWith_CharSequenceString_1() {
+		val obj1 = "abcdef"
+		should_startWith(null as String, null as String).assertFalse
+		should_startWith(obj1, null as String).assertFalse
+		should_startWith(null as String, obj1).assertFalse
+	} 
+
+	@Test
+	def should_endWith_CharSequenceString_0() {
+		val obj1 = "abcdef"
+		val obj2 = "abcdef"
+		val obj3 = "ghi"
+		val obj4 = "zf"
+		val obj5 = "ef"
+		should_endWith(obj1, obj1).assertTrue
+		should_endWith(obj1, obj2).assertTrue
+		should_endWith(obj1, obj3).assertFalse
+		should_endWith(obj1, obj4).assertFalse
+		should_endWith(obj1, obj5).assertTrue
+	}
+
+	@Test
+	def should_endWith_CharSequenceString_1() {
+		val obj1 = "abcdef"
+		should_endWith(null as String, null as String).assertFalse
+		should_endWith(obj1, null as String).assertFalse
+		should_endWith(null as String, obj1).assertFalse
+	}
+
+	@Test
+	def should_iterate_IteratorObject_0() {
+		val obj1 = #[ "a", "b", "c" ]
+		val obj2 = #[ "a", "b", "c" ]
+		val obj3 = #[ "a", "c", "b" ]
+		val obj4 = #[ "a", "b" ]
+		val obj5 = #[ "a", "b", "c", "d" ]
+		should_iterate(obj1.iterator(), obj1).assertTrue

```

Comment:
On a lot of these ``iterator`` calls, ``()`` is used. Is that intentional? It doesn't match up with the lack of ``()`` on all the calls to ``assertTrue``.